deployment/docker/sandbox/nginx.conf

```diff
@@ -166,7 +166,7 @@ http {
     location /status {
       include nginx-cors.conf;
 
-      # change proxy_pass to http://gateway:8080/overal/status;
+      # change proxy_pass to http://gateway:8080/overall/status for newer version;
       proxy_pass                    http://gateway:8080/status;
```

Comment:
change this to `proxy_pass                    http://gateway:8080/overall/status;`